t/e2e/whisper/whisper_mailbox_test.go

```diff
@@ -276,6 +276,31 @@ func (s *WhisperMailboxSuite) TestRequestMessagesInGroupChat() {
 	s.Require().Equal(helloWorldMessage, messages[0]["payload"].(string))
 }
 
+func (s *WhisperMailboxSuite) TestSendMessageWithoutSubscription() {
+	aliceBackend, stop := s.startBackend("alice")
+	defer stop()
+
+	// we need to wain >= whisper.DefaultSyncAllowance seconds to update
+	time.Sleep(12 * time.Second)
+
+	//get whisper service
```

Comment:
More care for comment quality. E.g.

```
// Retrieve the whisper service for Alice.
```

Reply:
Will fix all the comments in whisper tests.

Reply:
Fixed. Comments should be cleaner now.